packages/wallet-sdk/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@coinbase/wallet-sdk",
-  "version": "3.7.0",
+  "version": "3.7.1",
```

Comment:
fyi, `3.6.6` was published off from v3.6 branch https://github.com/coinbase/coinbase-wallet-sdk/tree/v3.6
this PR is to update version on master branch